sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslUdfUdafTest.java

```diff
@@ -90,6 +90,19 @@ public void testUdf() throws Exception{
                    BeamSql.query(sql2).registerUdf("cubic2", new CubicIntegerFn()));
     PAssert.that(result2).containsInAnyOrder(row);
 
+    String sql3 = "SELECT f_int, substr(f_string) as sub_string FROM PCOLLECTION WHERE f_int = 2";
+    PCollection<Row> result3 =
+        PCollectionTuple.of(new TupleTag<>("PCOLLECTION"), boundedInput1)
+            .apply("testUdf3",
+                   BeamSql.query(sql3).registerUdf("substr", UdfFnWithDefault.class));
+
+    RowType subStrRowType = RowSqlType.builder()
+        .withIntegerField("f_int")
+        .withIntegerField("sub_string")
```

Comment:
s/withIntegerField/withStringField/g

and I wonder why the UnitTest can pass while the data type is not right.